lib/virt_autotest/utils.pm

```diff
@@ -105,23 +105,56 @@ sub is_xen_host {
     return get_var("XEN") || check_var("SYSTEM_ROLE", "xen") || check_var("HOST_HYPERVISOR", "xen") || check_var("REGRESSION", "xen-hypervisor");
 }
 
-#check host to make sure it works well
+#check kernels by grep keywords from journals
 #welcome everybody to extend this function
-sub check_host {
+sub check_failures_in_journal {
+    my $machine //= 'localhost';
+
+    my $all_failures = "";
+    my @warnings = ('Started Process Core Dump', 'Call Trace');
+    foreach my $warn (@warnings) {
+        my $cmd = "journalctl | grep '$warn'";
+        $cmd = "ssh root\@$machine " . "\"$cmd\"" if $machine ne 'localhost';
+        if (script_run($cmd) == 0) {
+            if (get_var('KNOWN_KERNEL_BUG')) {
+                my $failure = "$warn on $machine: " . get_var('KNOWN_KERNEL_BUG');
+                $all_failures .= $failure;
+                record_soft_failure("$failure (Please ignore the 'fake bsc#5555' that has to be written here due to the silly PR code rule)");
```

Comment:
Why still need fake bug number is there is KNOWN_KERNEL_BUGS.

Reply:
`due to the silly PR code rule` in ci check in PR. I guess some patterns like "record_soft_failure.*bsc#NNN" is the rule.